conftest.py

```diff
@@ -64,20 +64,33 @@ def pytest_runtest_makereport(item, call):
     setattr(item, "rep_" + rep.when, rep)
 
 
-@pytest.fixture(autouse=True)
-def get_task_logs_on_failure(request):
-    """ Scheduler should be the only task running as root
-    """
-    yield
+def get_rotating_task_log_lines(task_id: str, task_file: str):
+    rotated_filenames = [task_file,]
+    rotated_filenames.extend(['{}.{}'.format(task_file, i) for i in range(1, 10)])
```

Comment:
The lines aren't appended, each log file is downloaded as a separate file. `9` was just the number in the ticket. If we find we need more we can change it, or, perhaps, add a call to look up how many rotated log files already exist.